Bottom: b74ab0c7f66e17b1e2eb42cbefefd163ef480592
Top:    54043291b57ab1cba8f17578c9e7d842d93dd363
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-24 16:24:01 +0800

Refresh of autofs-5.0.7-add-missing-libtirpc-lib-to-mount_nfs.so-when-TIRPC-enabled.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 9755e25..72d9135 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -60,6 +60,7 @@
 - fix typo forced-shutdown should be force-shutdown.
 - fix hesiod check error and use correct $(LIBS) setting.
 - fix dead LDAP symbolic link when LDAP support is disabled.
+- add missing libtirpc lib to mount_nfs.so when TIRPC enabled.
 
 25/07/2012 autofs-5.0.7
 =======================
